utils/economic-checks/src/util.rs

```diff
@@ -222,6 +249,27 @@ pub(crate) fn new_test_ext(
     .assimilate_storage(&mut t)
     .unwrap();
 
+    #[cfg(all(not(feature = "gear-native"), feature = "vara-native"))]
+    SessionConfig {
+        keys: initial_authorities
+            .iter()
+            .map(|x| {
+                (
+                    x.0.clone(),
+                    x.0.clone(),
+                    SessionKeys {
+                        babe: x.2.clone(),
+                        grandpa: x.3.clone(),
+                        im_online: x.4.clone(),
+                        authority_discovery: x.5.clone(),
+                    },
+                )
+            })
+            .collect::<Vec<_>>(),
+    }
+    .assimilate_storage(&mut t)
+    .unwrap();
```

Comment:
lead unwraps to expects, please

Reply:
This is just a test: `unwrap` vs `expect` makes practically no difference. We use `unwrap` everywhere in test externalities builder throughout the codebase (so do they in Substrate where this piece of code originally came from)